packages/cli/generators/model/index.js

```diff
@@ -297,12 +296,6 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
         if (this.artifactInfo.allowAdditionalProperties) {
           Object.assign(this.artifactInfo.modelSettings, {strict: false});
         }
-
-        this.log(
-          `Let's add a property to ${chalk.yellow(
-            this.artifactInfo.className,
```

Comment:
This change seems unrelated to the rest of this pull request, what is the reasoning for removing this message?

Reply:
Removed by accident, will add back 👍